Example/DApp/Sign/Accounts/AccountsViewController.swift

```diff
@@ -34,19 +38,51 @@ final class AccountsViewController: UIViewController, UITableViewDataSource, UIT
         super.viewDidLoad()
         navigationItem.title = "Accounts"
 
+
         navigationItem.rightBarButtonItem = UIBarButtonItem(
             title: "Disconnect",
             style: .plain,
             target: self,
             action: #selector(disconnect)
         )
+
+        navigationItem.leftBarButtonItem = UIBarButtonItem(
+            title: "Push Test",
+            style: .plain,
+            target: self,
+            action: #selector(pushTest)
+        )
+
         accountsView.tableView.dataSource = self
         accountsView.tableView.delegate = self
         session.namespaces.values.forEach { namespace in
             namespace.accounts.forEach { account in
                 accountsDetails.append(AccountDetails(chain: account.blockchainIdentifier, methods: Array(namespace.methods), account: account.address)) // TODO: Rethink how this info is displayed on example
             }
         }
+        proposePushSubscription()
+    }
+
+    func proposePushSubscription() {
+        let account = session.namespaces.values.first!.accounts.first!
+        // temp solution
+        let pairingTopic = Pair.instance.getPairings().last!.topic
```

Comment:
how it must work without temp solution?

Reply:
session object will contain pairing topic, I opened an issue for that already